app/src/main/java/ch/epfl/sweng/erpa/operations/RemoteServicesProviderCoordinator.java

```diff
@@ -26,29 +26,31 @@
 /**
  * This is responsible of memoizing and proxying the selected RemoteServicesProvider.
  * This class' life cycle is managed by Toothpick.
- *
+ * <p>
  * Hereafter rsp = Remote Services Provider.
- *
+ * <p>
  * This class implements the Proxy Pattern for RemoteServicesProvider.class.
  */
 @Singleton
 public class RemoteServicesProviderCoordinator implements InvocationHandler,
-                                                          DependencyConfigurator<RemoteServicesProvider> {
+        DependencyConfigurator<RemoteServicesProvider> {
+    private final String REMOTE_PROVIDER_KEY;
+    private final Context applicationContext;
     /*
      * A word or wary: This singleton may be called prior to an activity creation and may start
      * SelectRemoteServicesProviderActivity. Be conservative about injecting stuff here since it
      * may lead to an infinite dependency loop.
      */
```

Comment:
This comment should be the first thing on the class definition, please move the lines above after the comment